src/main/java/TechJobs.java

```diff
@@ -120,6 +120,23 @@ private static String getUserSelection(String menuHeader, HashMap<String, String
     // Print a list of jobs
     private static void printJobs(ArrayList<HashMap<String, String>> someJobs) {
 
-        System.out.println("printJobs is not implemented yet");
+        if (someJobs.size() < 1) {
+            System.out.println("No Results");
```

Comment:
```suggestion
            System.out.print("No Results");
```
I believe one of the unit tests fail on GitHub is due to spacing, which is kind of wonky for the test to count as failing (in my opinion; esp. for this line). So, I would recommend using `print()` instead of `println()` so in it doesn't print a newline after No Results.